lib/java/poi-wrapper/src/main/java/org/apache/poi/xssf/binary/XSSFBSheetHandler.java

```diff
@@ -66,8 +90,7 @@ public XSSFBSheetHandler(
     this.styles = styles;
     this.comments = comments;
     this.stringsTable = strings;
-    this.handler = sheetContentsHandler;
-    this.dataFormatter = dataFormatter;
+    this.handler = new XSSFBSheetContentsHandlerWrapper(sheetContentsHandler, dataFormatter);
```

Comment:
- wrapping `XSSFSheetXMLHandler.SheetContentsHandler` and `DataFormatter`
- into implementation of the new `XSSFBSheetContentsHandler` interface
- is fine way to refactor the code towards the new functionality
- tests in the POI repository will guarantee compatibility